terraform/scripts/requirements.txt

```diff
@@ -1,9 +1,7 @@
-boto3
-botocore
+boto3==1.12.10
 jmespath
 setuptools
```

Comment:
add versions

Reply:
Added version to jmespath and skipped version number to setuptools as the version depends on the version of Python (the minor version) and it is installed automatically